cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CloudProviderDTO.java

```diff
@@ -113,11 +113,19 @@ public void setUsernames( Map<OSType, String> usernames )
         this.usernames = usernames;
     }
 
+    /**
+     * @deprecated unsupported
+     */
+    @Deprecated
     public String getUsername()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)